x-pack/plugins/reporting/public/management/report_listing.tsx

```diff
@@ -325,28 +303,7 @@ class ReportListingUi extends Component<Props, State> {
       this.setState(() => ({
         isLoading: false,
         total,
-        jobs: jobs.map(
```

Comment:
The `map` is no longer needed as `apiClient.list` returns data in a usable format